mne_bids/tests/test_write.py

```diff
@@ -121,6 +121,45 @@ def _test_anonymize(raw, bids_path, events_fname=None, event_id=None):
     return bids_root
 
 
+def test_write_participants():
+    """Test that participants data is written correctly."""
```

Comment:
aren't there tests for this elsewhere already.

Quickly looked and found this: https://github.com/mne-tools/mne-bids/blob/409dd36fa1ce422b30b599e26adb09c771818d8d/mne_bids/tests/test_write.py#L383-L396

should we be consolidating these?

Reply:
I think it's okay to have the new test function, but maybe change its name and description to reflect that it's specifically meant to check for the issue you're fixing here. And these lines should go:

```python
    # assert age of participant is correct
     participants_tsv = op.join(bids_root, 'participants.tsv')
     data = _from_tsv(participants_tsv)
     assert data['age'][data['participant_id'].index('sub-01')] == '1'
```

Reply:
@jasmainak But I agree with you that we should try to rework, consolidate, clean up, and declutter our tests in general…

Reply:
I put the participants unit testing code from awhile back into this new unit test function to consolidate testing for participants.tsv file.